GameServerLib/Logic/Chatbox/Commands/ReloadLuaCommand.cs

```diff
@@ -10,6 +10,7 @@ public class ReloadLuaCommand : ChatCommand
 
         public override void Execute(Peer peer, bool hasReceivedArguments, string arguments = "")
         {
+            /*
```

Comment:
I think it would be cool if you get this command working with the name `ReloadScripts`